pennylane_lightning/core/src/measurements/MeasurementsBase.hpp

```diff
@@ -65,6 +67,23 @@ template <class StateVectorT, class Derived> class MeasurementsBase {
         : _statevector{statevector} {};
 #endif
 
+    /**
+     * @brief Set the seed of the internal random generator
+     *
+     * @param seed Seed
+     */
+    void setSeed(const size_t seed) { rng.seed(seed); }
```

Comment:
Nice! So the seed has to be set on each measurement object right?

Reply:
That's the current way in LQubit or GPU implementations (set randomly).